AK/StringView.cpp

```diff
@@ -299,4 +299,12 @@ Vector<StringView> StringView::split_view_if(Function<bool(char)> const& predica
     return v;
 }
 
+ByteBuffer StringView::characters_with_null_termination() const
```

Comment:
I don't think there's a need for an API on StringView to convert it to a NUL terminated byte buffer. At least, not for the change to LibGfx you're trying to make.